README.md

```diff
@@ -1064,6 +1064,28 @@ ranges can also be defined with `intranet_cidr`, `tunnel_cidr`,
     Usually the problem is not as severe as there are only 2-4
     Availability zones per region.
 
+##### Dynamic IP ranges
+
+The IP range for a VPC can also be dynamically alocated with the `ip_space` and `vpc_cidr_size` options. For example:
+
```

Comment:
For clarity add example section name so its more obvious where these options go.